### DISCUSSION TITLE
STM32U Msc Host example

### DISCUSSION THREAD

**Discussion by mcuenthusiast** [hathach/tinyusb#2154] [Q&A]
Hi All,

Does any one have tried to port stm32 synopsys for host.

I saw some old pull request If some can share it with me then it will be great I can start taking reference or modify further.

@hathach is this pull request contains code for host or tested as PoC for host? https://github.com/hathach/tinyusb/pull/505

**Comment by hathach** [comment]
> @hathach is this pull request contains code for host or tested as PoC for host? https://github.com/hathach/tinyusb/pull/505

yes

**Comment by mcuenthusiast** [comment]
Hi @hathach ,

Thanks for the response was it for dwc2 Host? I couldn't find any changes related to that so asking that.

My purpose her is to develop the dwc2 host APIs.

Thanks in advance !!!

**Reply by hathach** [reply]
no, dwc2 host driver is not implemented yet

**Reply by mcuenthusiast** [reply]
Okay, Thanks
